nservicebus/hosting/override-hostid.md

```diff
@@ -25,3 +25,10 @@ There are scenarios where the rules used by NServiceBus to generate a `hostid` a
 Manual configuration is required when deployments may end up in different paths than previously deployed versions (e.g. using [Octopus Deploy](https://octopus.com/)). The `hostid` must remain the same across restarts unless the physical host has changed.
 
 snippet: HostIdFixer
+
+
+## Stable host identifiers when using Docker
```

Comment:
Do we need to mention this here https://docs.particular.net/monitoring/metrics/install-plugin#configuration-instance-id  as well?

Reply:
@andreasohlund it copies the ID from core.